source/_integrations/zwave_js.markdown

```diff
@@ -771,6 +771,10 @@ S0 security imposes significant additional traffic on your mesh and is recommend
 
 S2 security does not impose additional network traffic and provides additional benefits, such as detecting packet corruption. By default, Z-Wave attempts S2 security during inclusion if supported, falling back to S0 security only when necessary.
 
+#### Where can I see the security keys in the Z-Wave JS add-on?
+
+After the initial setup of the Z-Wave controller, you can view the security keys in the Z-Wave JS add-on. Go to {% my supervisor_addon addon="core_zwave_js" title="**Settings > **Add-ons** > **Z-Wave JS**" %} and open the **Configuration** tab. You can now see the network security key and the S0 and S2 security keys.
```

Comment:
Only 4 keys are relevant, the 3 S2 keys and the S0 key.
"Network key" is a legacy config setting, identical to the S0 key.

Reply:
thank you, @AlCalzone :+1: . Updated section.